pkg/aws/client.go

```diff
@@ -344,18 +351,33 @@ func (c *awsClient) CheckStackReadyOrNotExisting(stackName string) (stackReady b
 				return true, nil
 			}
 			if *summary.StackStatus != cloudformation.StackStatusDeleteComplete {
-				return false, fmt.Errorf("Error creating user: Cloudformation stack %s exists with status %s. "+
+				return false, fmt.Errorf("Error creating CloudFormation Stack: Cloudformation stack %s exists with status %s. "+
 					"Expected status is %s.\n"+
-					"Ensure user osdCcsAdmin does not exist, then retry with\n"+
+					"Ensure user osdCcsAdmin CloudFormation Stack does not exist, then retry with\n"+
 					"moactl init --delete-stack; moactl init",
 					*summary.StackName, *summary.StackStatus, cloudformation.StackStatusCreateComplete)
 			}
 		}
 	}
-
 	return false, nil
 }
 
+func (c *awsClient) CheckAdminUserNotExisting(userName string) (err error) {
+	userList, err := c.iamClient.ListUsers(&iam.ListUsersInput{})
+	if err != nil {
+		return err
+	}
+	for _, user := range userList.Users {
+		if *user.UserName == userName {
+			return fmt.Errorf("Error creating user: IAM user '%s' already exists."+
+				"Ensure user osdCcsAdmin IAM user does not exist, then retry with\n"+
+				"moactl init --delete-stack; moactl init",
```

Comment:
I don't think this function should advise the user to delete the stack. If we determine that the user exists, and that the stack is ready (we get this from the previous function call) then the user existing should return success

Reply:
If the CF stack is ready then this function shouldn't be called. 
This function will only be called prior to creating a new stack, hence the need to verify there is no IAM user with this name.

Reply:
I removed the part that asks for `moactl init --delete-stack` from the error.